src/vm/proftoeeinterfaceimpl.cpp

```diff
@@ -996,7 +1005,11 @@ ClassID SafeGetClassIDFromObject(Object * pObj)
     {
         NOTHROW;
         GC_NOTRIGGER;
+#if defined(PROFILING_SUPPORTED) // TODO - This inline function is only defined whith
+                                 // PROFILING_SUPPORTED. Fix this when separating
+                                 // Profiling and ETW
```

Comment:
Delete this line completely instead of ifdefing it out ... it is low-value questionable debug-only check.